.github/workflows/CI.yaml

```diff
@@ -45,14 +45,23 @@ jobs:
         df -h
         ulimit -a
 
+    - name: Determine Conda env file
```

Comment:
This could be simplified by making the file names the same as the `matrix.os` variable like: `environment-file: devtools/conda-envs/asapdiscovery-${{ matrix.os }}.yaml`

Reply:
good idea, change made.